rpcs3/Emu/RSX/Program/GLSLCommon.cpp

```diff
@@ -764,7 +770,7 @@ namespace glsl
 			OS <<
 
 #ifdef __APPLE__
-			"vec4 remap_vector(const in vec4 rgba, const in uint remap_bits)\n"
+			"vec4 remap_vector_m(const in vec4 rgba, const in uint remap_bits)\n"
```

Comment:
Why was this renamed?
Just wondering.

Reply:
It conflicted with the other remap_vector (both are required on macOS).